src/js/playlist/preload.js

```diff
@@ -0,0 +1,10 @@
+const _ = require('utils/underscore');
+
+const preloadValues = ['none', 'metadata', 'auto'];
```

Comment:
If you use an object you don't need to do `_.contains`. An es6 `Map` is even better if we support it.